modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor_balloon.jsp

```diff
@@ -32,6 +38,8 @@ String name = namespace + GetterUtil.getString((String)request.getAttribute(CKEd
 				"contents", contents
 			).put(
 				"name", HtmlUtil.escapeAttribute(name)
+			).put(
+				"onChangeMethodName", HtmlUtil.escapeJS(onChangeMethod)
```

Comment:
I am not sure if `HtmlUtil.escapeJS` is right here. Doesn't it mess up the string if used in tag attribute? I think the React component may get gibberish value for method name.

Reply:
> Doesn't it mess up the string if used in tag attribute? I think the React component may get gibberish value for method name.

Could you provide an example which type of value our React component can receive? I copied this code from [classic-editor tag](https://github.com/liferay/liferay-portal/blob/379614c23aafe7d1373a662f542d92fab91423c2/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/ckeditor_classic.jsp#L51)

Reply:
Oh sorry, fake alert! I just doublechecked it and it works fine. I was thinking of `jsonSerializer.serializeDeep`, that I needed to remove in https://github.com/liferay/liferay-portal/commit/b79f3601cbd0e603868fd54d5ea1ad90284b4f14